.mvn/gradle-enterprise-custom-user-data.groovy

```diff
@@ -17,11 +17,17 @@ if (System.env.GITHUB_ACTIONS) {
     buildScan.value('gh-actor', System.env.GITHUB_ACTOR)
     buildScan.value('gh-workflow', System.env.GITHUB_WORKFLOW)
 
+    if (System.env.GITHUB_EVENT_NAME == "pull_request" && System.env.GITHUB_EVENT_PATH != null) {
```

Comment:
don't we have cases where jobs are triggered by a pullrequest as a workflow request where this info won't exist?

Reply:
I have no idea what you're saying :). Could you explain which case you are trying to cover?

Reply:
We discussed it with Max, what I did is fine and this particular case has nothing to do with PR execution but withe event payloads sent to the bot webhook.